gg-pac-team3-front/src/App.jsx

```diff
@@ -1,34 +1,17 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
+import { BrowserRouter, Routes, Route} from 'react-router-dom';
+import FrontPage from './pages/frontPage/frontpage';
```

Comment:
This import does not work for me. When I run the project, I get an error in the browser that it cannot find this import. When I change it to `./pages/frontPage/FrontPage` it works, which is the name of the component rather than the name of the file.

Reply:
Got you! sorry lots of file changes were made last night.